mypy/types.py

```diff
@@ -27,6 +27,19 @@
 
 JsonDict = Dict[str, Any]
 
+# The set of all valid expressions that can currently be contained
```

Comment:
Add `TODO` comments here about some things? At least enum support is probably not going to happen immediately, so maybe it should be as a `TODO` comment?